helper_test.go

```diff
@@ -577,6 +577,44 @@ func createWorkspace(t *testing.T, client *Client, org *Organization) (*Workspac
 	}
 }
 
+func createWorkspaceWithVCS(t *testing.T, client *Client, org *Organization) (*Workspace, func()) {
+	var orgCleanup func()
+
+	if org == nil {
+		org, orgCleanup = createOrganization(t, client)
+	}
+
+	ctx := context.Background()
+
+	token, ocTestCleanup := createOAuthToken(t, client, org)
+	w, err := client.Workspaces.Create(ctx, org.Name, WorkspaceCreateOptions{
+		Name: String(randomString(t)),
+		VCSRepo: &VCSRepoOptions{
+			OAuthTokenID: String(token.ID),
+		},
+	})
+
+	if err != nil {
```

Comment:
_nitting_ we usually don't insert a blank line between the call and the error check